docs/examples.md

```diff
@@ -32,3 +34,109 @@
 ### [Basic setup w/ Typescript](../examples/basic-setup-typescript)
 ![typings](./assets/tooling-typings-vs-code.png)
 ![examples-basic](https://cloud.githubusercontent.com/assets/591643/24778192/1f0684ec-1ade-11e7-866b-b11bb60ac109.png)
+
+
+## Running the examples from scratch
+
+
+### Step 0: Gather all your tools
+
+Before getting started with react-sketch-app you should have all the following tools installed and ready to go:
+
+- [ ] A terminal program (if you are not very comfortable using the command line, [this guide](http://wiseheartdesign.com/articles/2010/11/12/the-designers-guide-to-the-osx-command-prompt/) should teach you most of the basic commands to get up and running)
+
+- [ ] Sketch
+
+- [ ] A Text editor ([Atom](https://atom.io/) or [Sublime Text](https://www.sublimetext.com/) are straightforward choices with which to start and both have many of tutorials available online)
+
+- [ ] A recent version of NodeJS/NPM (If you type ```node -v``` in your terminal and do not see a version number go ahead and install it from the official [Node.js® website](https://nodejs.org), the button with "LTS" should be exactly what you need)
+
+- [ ] Git (If you type ```git --version``` in your terminal and do not see a version number go ahead and install git for your operating system from the [git website](https://git-scm.com/downloads))
+
+### Step 1: Clone react-sketchapp
+
+Assuming all your tools are installed/working, we're ready to get started by getting ```react-sketchapp``` onto your machine
+
+- [ ] Open a terminal and change the directory into a folder where you want react-sketchapp to live (if you're not sure or don't really care, an easy choice is to put it right on your desktop. You can move your terminal inside there by typing ```cd ~/Desktop``` & verify you're inside that directory by typing ```pwd``` to see the full path. ```cd ..``` will move you back one folder)
+
+- [ ] You now can clone or download react-sketchapp to the directory from the step above by copying the following command into your terminal:
+
+```sh
+git clone https://github.com/airbnb/react-sketchapp.git
```

Comment:
it's a bit going against the instructions given in each examples: curl each example separetely so the people are not scared by all the other files

Reply:
I suppose it does break the convention slightly, however, my sense is that just to get them up and running it's a much cleaner approach to pull in the entire repo. Each of the individual examples have a line that say "Download the example or clone the repo" and the distinction might be lost on new folks. If they run through the instructions they'll probably pick up how to navigate with the command line and can jump off from there.

 Then individual clone commands with pipes to things like ``` | tar -xz --strip=2 ``` or ```curl``` would make more sense

Reply:
@mathieudutour What are you thinking?